bgstally/windows/fleetcarrier.py

```diff
@@ -83,8 +93,11 @@ def show(self):
 
         current_row += 1
 
-        ttk.Label(frm_items, text=_("Selling Commodities"), font=FONT_HEADING_2).grid(row=current_row, column=0, sticky=tk.W) # LANG: Label on carrier window
-        ttk.Label(frm_items, text=_("Buying Commodities"), font=FONT_HEADING_2).grid(row=current_row, column=1, sticky=tk.W) # LANG: Label on carrier window
+        ttk.Checkbutton(frm_items, text=_("Selling Commodities"), style='This.TCheckbutton', variable=self.bgstally.state.FcSellingCommodities, onvalue=CheckStates.STATE_ON, \
+                        offvalue=CheckStates.STATE_OFF, command=partial(self.bgstally.state.FcSellingCommodities.set)).grid(row=current_row, column=0, padx=10, sticky=tk.W)
+
+        ttk.Checkbutton(frm_items, text=_("Buying Commodities"), style='This.TCheckbutton', variable=self.bgstally.state.FcBuyingCommodities, onvalue=CheckStates.STATE_ON, \
+                        offvalue=CheckStates.STATE_OFF, command=partial(self.bgstally.state.FcBuyingCommodities.set)).grid(row=current_row, column=1, padx=10, sticky=tk.W)
```

Comment:
_This makes me think I should be moving to ttk styles throughout :)  My problem, not yours!_

_We also need translation hints for these lines, like this: ` # LANG: Label on carrier window` but my hacked-together hit extractor can't cope with continuation lines using `\` where the translatable string is on a different line to the hint.  Also my problem, not yours._